packages/@webex/plugin-meetings/src/mediaQualityMetrics/config.ts

```diff
@@ -4,6 +4,11 @@ export const emptyMqaInterval = {
   intervalMetadata: {
     peerReflexiveIP: '0.0.0.0',
     peripherals: [],
+    cpuInfo: {
+      numberOfCores: 1, // default value from spec if CpuInfo.getNumLogicalCores cannot be determined
+      description: 'web client',
```

Comment:
According to [types/CpuInfo.raml](https://sqbu-github.cisco.com/WebExSquared/event-dictionary/blob/main/types/CpuInfo.raml), description is  -> "This is a free-form text string representation of the CPU".  However here we are using "web client".  If we have put architecture as undefined maybe we can put desc as undefined as well.

Reply:
@Shreyas281299 It's a valid point. I checked the spec again and it's an enum, so the value 'unknown' seems logical to me too here. I recall that this value was suggested during a conversation with the MQA team, but I will verify with them and hopefully, they will also agree. Thanks

Reply:
@Shreyas281299 I discussed this with the MQA team, and I put an `NA` value that works for them. Since we can't omit this value, it will be a more suitable string value.

Reply:
Sounds good.